src/create-glamorous.js

```diff
@@ -7,11 +7,17 @@ import {PropTypes} from './react-compat'
 import withTheme from './with-theme'
 import getGlamorClassName from './get-glamor-classname'
 
+let glamorousInstance = 1
+
 export default createGlamorous
 
 function createGlamorous(splitProps) {
   // TODO: in a breaking version, make this default to true
-  glamorous.config = {useDisplayNameInClassName: false}
+  glamorous.config = {
+    useDisplayNameInClassName: false,
+    idCounter: 1,
+    instanceId: glamorousInstance++,
```

Comment:
you should heavily document good SSR patterns here; it _can_ work, but requires the server to be careful with managing glamorous instances